astropy/modeling/physical_models.py

```diff
@@ -73,7 +73,7 @@ class BlackBody(Fittable1DModel):
     _input_units_allow_dimensionless = True
 
     # We enable the spectral equivalency by default for the spectral axis
-    input_units_equivalencies = {"x": u.spectral()}
+    input_units_equivalencies = {'x': u.spectral()}
```

Comment:
Is this hardcoding here okay?

Reply:
Yes, that will have to do. I don't know how to get to the names of inputs at that level.